src/Core/Context.idr

```diff
@@ -1118,13 +1121,13 @@ getFieldNames ctxt recNS
 -- Find similar looking names in the context
 export
 getSimilarNames : {auto c : Ref Ctxt Defs} ->
-                   Name -> Core (Maybe (String, List (Name, Visibility, Nat)))
+                   Name -> Core (Maybe (String, List (Name, WithDefault Visibility Private, Nat)))
```

Comment:
I think these can be collapsed eagerly. We will never care about anything but the actualy visibility here.